src/cmd_line/commands/normal.ts

```diff
@@ -0,0 +1,40 @@
+import { Parser, all, seq, whitespace } from 'parsimmon';
+import { VimState } from '../../state/vimState';
+import { ExCommand } from '../../vimscript/exCommand';
+import { LineRange } from '../../vimscript/lineRange';
+
+export class NormalCommand extends ExCommand {
+  public static readonly argParser: Parser<NormalCommand> = seq(whitespace, all).map(
```

Comment:
Add a `TODO` regarding `:normal!`

Reply:
Fix it.
[3643304](https://github.com/VSCodeVim/Vim/pull/8896/commits/3643304f2b96e325d261b9b37e769889e1686c72)